data/json/recipes/recipe_food.json

```
@@ -29,7 +29,7 @@
     "skill_used": "cooking",
     "difficulty": 3,
     "time": "30 m",
-    "autolearn": true,
+    "book_learn": [ [ "baking_book", 2 ] ],
```

```suggestion
    "book_learn": [ [ "cookbook_foodfashions", 5 ], [ "baking_book", 2 ] ],
```
Based on these being a favourite at my local trendy food place.